src/who-we-are/index.md

```diff
@@ -32,6 +31,15 @@ offers to make services better for the people who depend on them. And,
 ultimately, to free up more resources for essential front line services like
 education, health and social care, housing, welfare, and policing.
 
+## Our mission
+
+dxw's mission is to:  
+
+1. build services that fit seamlessly into user’s lives, and that make public services usable and accessible to all – especially those most in need  
```

Comment:
This one has maybe a few to many clauses.
How about:
1. build public services that fit seamlessly into people’s lives, and everyone can access and use – especially those most in need